### PR TITLE
DOCS-1276: Reorder boards

### PR DESCRIPTION
**Pull request by andf-viam**
Reorder boards in board carousel on index page per updated ordering from Jon, with SolEng input.
- Kept order exactly as requested, then listing "no preference" boards together by vendor afterwards, otherwise ordered by previous order.

### CODE REVIEW

File: docs/components/board/_index.md [hunk 1]
```diff
@@ -36,19 +36,36 @@ A board can be:
...
+    <li><a href="#c1_slide7">NVIDIA Jetson Nano</a></li>
+    <li><a href="#c1_slide8">NVIDIA Jetson AGX Orin</a></li>
+    <li><a href="#c1_slide9">NVIDIA Jetson Xavier NX</a></li>
+    <li><a href="#c1_slide10">Raspberry Pi 3</a></li>
+    <li><a href="#c1_slide11">Raspberry Pi Zero 2W</a></li>
+    <li><a href="#c1_slide12">BeagleBone AI-64</a></li>
+    <li><a href="#c1_slide13">Numato GPIO Modules</a></li>
+    <li><a href="#c1_slide14">PCA9686 I<sup>2</sup>C Interface</a></li>
+
+
```
Comment (npentrel) [inline docs/components/board/_index.md:55]:
```suggestion
```

File: docs/components/board/_index.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ For model-specific configuration information, click on one of the following mode
...
 | [`numato`](numato/) | [Numato GPIO Modules](https://numato.com/product-category/automation/gpio-modules/), peripherals for adding GPIO pins |
 | [`pca9685`](pca9685/) | [PCA9685 Arduino I<sup>2</sup>C Interface](https://www.adafruit.com/product/815), a 16-channel [I<sup>2</sup>C](#i2cs) [PWM](https://docs.arduino.cc/learn/microcontrollers/analog-output)/[servo](/components/servo/) driver peripheral |
-| [`upboard`](upboard/) | An Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) |
-| [`fake`](fake/) | A model used for testing, with no physical hardware |
 | [`customlinux`](customlinux/) | A model for other Linux boards. |
 | other | You can use other boards with modular components such as [`periph_board`](https://github.com/viam-labs/periph_board) |
```
Comment (andf-viam) [inline docs/components/board/_index.md:50]: Aside: we could remove `other` from this list, in favor of the new direct listing of MR resources in the next header, which includes `periph_board`.

Comment (npentrel) [reply docs/components/board/_index.md:50]: can you make a ticket for that? I think we need to ask to make sure

**Review by npentrel — APPROVED** [review]

File: docs/components/board/_index.md (hunk 2 above)
Comment (andf-viam) [reply docs/components/board/_index.md:50]: Created DOCS-1280 to consider. 👍

**Pull request merged**